rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1034,28 +1072,35 @@ private void allowFederationElectionAgain() {
 
     private boolean svpIsOngoing() {
         return federationSupport.getProposedFederation()
-            .map(Federation::getCreationBlockNumber)
-            .map(proposedFederationCreationBlockNumber ->
-                proposedFederationCreationBlockNumber + bridgeConstants.getFederationConstants().getValidationPeriodDurationInBlocks())
-            .filter(validationPeriodEndBlock -> rskExecutionBlock.getNumber() <= validationPeriodEndBlock)
+            .filter(this::validationPeriodIsOngoing)
             .isPresent();
     }
 
-    protected void processSvpFundTransactionUnsigned(Transaction rskTx) throws IOException, InsufficientMoneyException {
-        Optional<Federation> proposedFederationOpt = federationSupport.getProposedFederation();
-        if (proposedFederationOpt.isEmpty()) {
-            String message = "Proposed federation should be present when processing SVP fund transaction.";
-            logger.warn(message);
-            throw new IllegalStateException(message);
-        }
-        Federation proposedFederation = proposedFederationOpt.get();
+    private boolean validationPeriodIsOngoing(Federation proposedFederation) {
+        long validationPeriodEndBlock = proposedFederation.getCreationBlockNumber() +
+            bridgeConstants.getFederationConstants().getValidationPeriodDurationInBlocks();
 
-        Coin spendableValueFromProposedFederation = bridgeConstants.getSpendableValueFromProposedFederation();
-        BtcTransaction svpFundTransactionUnsigned = createSvpFundTransaction(proposedFederation, spendableValueFromProposedFederation);
+        return rskExecutionBlock.getNumber() <= validationPeriodEndBlock;
```

Comment:
Why is the block `validationPeriodEndBlock` still considered ongoing? Should not this logic be equal to the migration age? [There](https://github.com/rsksmart/rskj/blob/a13a0458f3db13928a5f15d4048503e7cb5f5c2b/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L1266), we are not considering the end block an ongoing state.

Reply:
I would suggest going with the same logic as for migration to be consistent with the meaning of the end block of an ongoing process.

Reply:
makes sense

Reply:
updated

Reply:
Nice!